WalletWasabi.Fluent/ViewModels/Wallets/Send/SendFeeViewModel.cs

```diff
@@ -38,22 +38,39 @@ public SendFeeViewModel(Wallet wallet, TransactionInfo transactionInfo, bool isS
 		SetupCancel(false, true, false);
 		EnableBack = true;
 
-		NextCommand = ReactiveCommand.Create(() =>
-	   {
-		   _transactionInfo.ConfirmationTimeSpan = TransactionFeeHelper.CalculateConfirmationTime(FeeChart.CurrentConfirmationTarget);
+		NextCommand = ReactiveCommand.Create(OnNext);
 
-		   Complete();
-	   });
+		AdvancedOptionsCommand = ReactiveCommand.CreateFromTask(ShowAdvancedOptionsAsync);
 	}
 
 	public FeeChartViewModel FeeChart { get; }
 
-	private void Complete()
+	public ICommand AdvancedOptionsCommand { get; }
+
+	private void OnNext()
 	{
-		var blockTarget = FeeChart.CurrentConfirmationTarget;
+		if (!_transactionInfo.IsCustomFeeUsed)
+		{
+			_transactionInfo.ConfirmationTimeSpan = TransactionFeeHelper.CalculateConfirmationTime(FeeChart.CurrentConfirmationTarget);
+
+			var blockTarget = FeeChart.CurrentConfirmationTarget;
+
+			Services.UiConfig.FeeTarget = (int)blockTarget;
+			Close(DialogResultKind.Normal, new FeeRate(FeeChart.GetSatoshiPerByte(blockTarget)));
+		}
+		else
+		{
+			Close(DialogResultKind.Normal, _transactionInfo.FeeRate);
+		}
```

Comment:
It needs to be discussed what to do in different cases...

1. Fee dialog opened, custom fee is not set. Done pressed.
   - The value from the fee slider should be returned.
   
2. Fee dialog opened, custom fee rate dialog opened and selected.
   - Custom fee rate should be returned.

3. Fee dialog opened, custom fee rate is set, Done pressed.
   - I think in this case the user intends to use the value from the fee slider.
   
What do you think?